modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -122,6 +122,62 @@ definition {
 		ObjectField.addAnyFolder(folderName = "Folder 151566");
 	}
 
+	@description = "LPS-159151 - Verify that it's possible to create an Action with a custom field in a System Object"
+	@priority = "4"
+	test CanCreateActionWithCustomFieldInSystemObjects {
+		task ("Add a Custom Object and Custom Field (API)") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 151573",
+				objectName = "CustomObject151573",
+				pluralLabelName = "Custom Objects 151573");
```

Comment:
Use the ticket number correct to add the Custom Object - [LPS-162194](https://issues.liferay.com/browse/LPS-162194)